okcupyd/looking_for.py

```diff
@@ -31,7 +31,8 @@ class LookingFor(object):
     profile.
     """
 
-    _ages_re = re.compile(u'Ages ([0-9]{1,3})\u2013([0-9]{1,3})')
+    _ages_re = re.compile(u'\s*Ages ([0-9]{1,3})\u2013([0-9]{1,3})\s*')
```

Comment:
I didn't read this closely. Is there any reason not to declare the encoding for this file to be utf-8 so that we can avoid the \u2013 and just put the actual unicode character there?